.changeset/calm-files-hunt.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-search-backend-module-elasticsearch': patch
+---
+
+Allow indexPrefix configuration through the app-config.yaml
```

Comment:
I suppose you could wrap indexPrefix and app-config.yaml in backticks to avoid this warning.